### PR TITLE
docs(blueprint composer): updated blueprint composer documentation

### PR DESCRIPTION
**Pull request by AlexanderKorotkov**

### CODE REVIEW

**Comment by guy-mograbi-at-gigaspaces** [comment]
we should add a section on how to install. 

it should document

 - given rpm, run this command to install it
 - once rpm is installed, run X command to run the composer
 - configuration: port, me.json etc.. options. (default values)

**Comment by nir0s** [comment]
Should I review?

**Comment by guy-mograbi-at-gigaspaces** [comment]
Yes. Thanks.

File: content/composer/Introduction.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (nir0s) [inline content/composer/Introduction.md]: this should go in the 'overview.md' file

File: content/composer/Introduction.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+The Blueprint Composer is a graphical editor for creating blueprint YAML files dynamically using
+a modern Drag and Drop Interface, including network items, custom types and inline types and
+custom plugins and interfaces. The composer enables topology modeling for complex
+applications while providing means to add relevant lifecycle operations implementation via
+external plugins and scripts. The end result will be a downloadable file generated by the
+composer which will provide a TOSCA based description for the application topology and
+lifecycle management.
+This guide will quickly walk you through the use of the Composer.
```
Comment (nir0s) [inline content/composer/Introduction.md]: which guide?

File: content/composer/how-to-install.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (nir0s) [inline content/composer/how-to-install.md]: please remove this page, use the current "installation.md" page instead and add the "change the default types" section to it.

File: content/composer/Introduction.md (hunk 1 above)
Comment (AlexanderKorotkov) [inline content/composer/Introduction.md]: done

File: content/composer/how-to-install.md (hunk 3 above)
Comment (AlexanderKorotkov) [inline content/composer/how-to-install.md]: done

File: content/composer/Introduction.md (hunk 2 above)
Comment (AlexanderKorotkov) [inline content/composer/Introduction.md]: Extra line, removed

**Comment by sivanbar** [comment]
@AlexanderKorotkov This is basically a rearrangement into files of the guide sent to Huawei, right?

File: content/composer/available-actions.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+---
+layout: bt_wiki
+title: Available Actions
+category: Docs
+draft: false
+weight: 300
+
+---
+
+On the top right you can find additional actions to perform on the current opened blueprint.
```
Comment (nir0s) [inline content/composer/available-actions.md]: current -> currently

**Pull request closed without merging**